Lab1/View Controllers/SingUpViewController.swift

```
@@ -0,0 +1,155 @@
+//
+//  SingUpViewController.swift
+//  Lab1
+//
+//  Created by Liubomyr on 9/19/19.
+//  Copyright © 2019 Liubomyr. All rights reserved.
+//
+
+import UIKit
+import FirebaseAuth
+import Firebase
+
+class SingUpViewController: UIViewController, UITextFieldDelegate {
+
+    @IBOutlet weak var name: UITextField!
+    @IBOutlet weak var number: UITextField!
+    @IBOutlet weak var email: UITextField!
+    @IBOutlet weak var password: UITextField!
+    @IBOutlet weak var signUpButton: UIButton!
+    @IBOutlet weak var errorLabel: UILabel!
+    static var idUser: String = ""
+    
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpElements()
+
+    }
+    func setUpElements()  {
+        
+        //Hide the error label
```

ну дивись ти написав комент //Hide the error label, але ти просто змінив alpha коли є пропертя isHidden - пиши комент що вони справді роблять ))) просто це трохи різні речі. Краще додай алер якщо шось впаде то показуй його